src/StartupManager/Commands/CommandBuilder.cs

```diff
@@ -95,5 +98,20 @@ private static Command GetEnableStartupCommand() {
 
             return enableCommand;
         }
+
+        private static Command GetSetPriorityCommand() {
+            var setPriorityCommand = new Command("set-priority") {
+                Description = "Sets the priority of a Task Scheduler startup program"
+            };
+
+            setPriorityCommand.AddAlias("sp");
```

Comment:
I am wondering if it would make more sense to use a single letter to be consistent with the others.. for instance `p` for priority. This would make it look more similar in the help output.

<img width="422" height="484" alt="Image" src="https://github.com/user-attachments/assets/3a599af3-4743-4c99-a1f0-d71b2e2af3e1" />